content/pl/concepts/overview/what-is-kubernetes.md

```diff
@@ -0,0 +1,94 @@
+---
+reviewers:
+- bgrant0607
+- mikedanese
+title: Kubernetes — co to jest?
+content_template: templates/concept
+weight: 10
+card:
+  name: concepts
+  weight: 10
+---
+
+{{% capture overview %}}
+Na tej stronie znajdziesz ogólne informacje o Kubernetes.
+{{% /capture %}}
+
+{{% capture body %}}
+Kubernetes to przenośna, rozszerzalna platforma oprogramowania *open-source* służąca do zarządzania zadaniami i serwisami uruchamianymi w kontenerach, która umożliwia deklaratywną konfigurację i automatyzację. Ekosystem Kubernetes jest duży i dynamicznie się rozwija. Serwisy Kubernetes, wsparcie i narzędzia są szeroko dostępne.
+
+Nazwa Kubernetes pochodzi z greki i oznacza sternika albo pilota. Google otworzyło projekt Kubernetes publicznie w 2014. Kubernetes korzysta z [piętnastoletniego doświadczenia Google w uruchamianiu wielkoskalowych serwisów](https://ai.google/research/pubs/pub43438) w połączeniu z najlepszymi ideami i praktykami wypracowanymi przez społeczność.
+
+## Trochę historii
+
+Cofnijmy sie trochę w czasie i zobaczmy, dlaczego Kubernetes jest taki przydatny.
+
+![Deployment evolution](/images/docs/Container_Evolution.svg)
```

Comment:
small nit: accessibility text not localized

Reply:
Fixed.